### PR TITLE
User permissions

### LINKED ISSUE #1310: Implement Account - User edit permissions

### PR DESCRIPTION
**Pull request by na3d**
Closes #1310

### CODE REVIEW

File: src/users/user/layouts/PermissionsPage.js [hunk 1]
```diff
@@ -1,7 +1,134 @@
...
+          title="NodeBalancer permissions"
+          section="nodebalancers"
+          addLabel="Can add NodeBalancers to this Account ($)"
+          addCheck={global.add_nodebalancers}
+        />
+        <PermissionCard
+          updateGlobal={this.updateGlobal}
+          title="DNS Manager permissions"
+          section="dnszones"
+          addLabel="Can add Domains using the DNS Manager"
```
Comment (mparke) [inline src/users/user/layouts/PermissionsPage.js]: `Domains` > `domains` ?

Comment (na3d) [inline src/users/user/layouts/PermissionsPage.js]: Keeping it the same as the current manager.  We can change it later with no problem later if need be.

**Comment by abemassry** [comment]
Not sure if this is in the scope
image

After selecting no and navigating away from the page, the expectation is that it would stay on no

**Comment by abemassry** [comment]
@eatonphil says this page may change and the radio might go away

**Pull request merged**